### DISCUSSION TITLE
Allow Dashboard access for external users

### DISCUSSION THREAD

**Discussion by Hutch79** [goauthentik/authentik#11658] [Q&A] [answered]
Hey everyone 👋 

I use Authentik 2024.8.2 and noticed that my External users can't access the Dashboard.

I would like my Users to be able to access the Dashboard so they have a central location where they see all applications which they have access to.

Which setting do I need to change to achieve this?
Thanks already for the help!

Edit:
Another thing that would be solved by this is the self-management for users.
Currently, they have no way to change their password except for the password reset flow.

**Comment by motey** [comment, marked as answer]
You have to migrate external Users to internal. There is no other solution. Had a long chat with the support.
If you are using the OpenSource Version you can just migrate your users, but if you are on an Enterprise subscription you first need to buy more internal licenses (Which can make substantial difference in costs just to use the Dashboard.)

As far as i can grasp it,  there is no self-management for external users anymore, which is definitive a feature gap or depending on your perspective also a security gap.

**Reply by Hutch79** [reply]
That sounds... weird?

The Dashboard is a pity, but no self-management for external users? That sounds just stupid to me...

Well, thanks anyway for the answer, I appreciate it!

**Reply by motey** [reply]
> That sounds... weird?

yea, right? Enterprise support was professional on the surface, but in the end communicated in the subtext that i am the nag not just accepting this feature rug pull. They avoided to communicated any reason why they did it. 
I mean just a "hey we are very low on funds and we think that is our only way to keep floating"-message would have done it. but this "Duh stupid, you just have to pay more! We dont need to tell you any reason!"
 At the moment i have no handle to justify the costs for enterprise support to my management next year. its a pity. I like the project.

**Reply by Hutch79** [reply]
I'm lucky since I'm only using it in my Home Lab, so I don't need the Enterprise features.
But I totally understand that it's hard to justify a solution like Authentik which will provide fewer features (no Enterprise subscription) than other alternatives like Keycloak. Management will not like this.
The Dashboard is details, but user self-management is a necessity.

I get the financial reasoning. It gets harder and harder for Open Source projects to fund themselves. We see it with Read Hat, Redis and co. But communication is key.

I guess there are only to ways to go now.
Accept it, use it and see where they go or switch to another solution like Keycloak or Zitadel. Although, Authentik is by far my favorite.
